submit-build-status/action.yml

```diff
@@ -33,17 +33,25 @@ runs:
       echo "User description: ${{ inputs.user_description }}"
 
   - name: Validate inputs
-    shell: python
+    shell: bash
+    env:
+      USER_REASON: "${{ inputs.user_reason }}"
+      USER_DESCRIPTION: "${{ inputs.user_description }}"
     run: |
-      if '${{ inputs.build_status }}' == '':
-        print('Need to specify a build status that is not empty!')
-        sys.exit(1)
-      if len('${{ inputs.user_reason }}') > 200:
-        print('Specified user reason has more than 200 characters!')
-        sys.exit(1)
-      if len('${{ inputs.user_description }}') > 1000:
-        print('Specified user description has more than 1000 characters!')
-        sys.exit(1)
+      if [ "${{ inputs.build_status }}" == "" ]; then
```

Comment:
why do we define the other inputs as env var but this one we template the gha way?
Maybe for consistency we could move it to be a env var as well?

Reply:
Yea can do that too for consistency.

Background: the check for an empty string also works when comparing to literals but determining the string length requires a variable beforehand.